src/sentry/search/events/filter.py

```diff
@@ -1101,6 +1102,10 @@ def resolve_params(self) -> List[WhereType]:
         if "start" not in self.params or "end" not in self.params:
             raise InvalidSearchQuery("Cannot query without a valid date range")
         start, end = self.params["start"], self.params["end"]
+        # Update start to be within retention
+        expired, start = outside_retention_with_modified_start(
+            start, end, self.params.get("organization_id")
```

Comment:
Actually - do we need to pass in the org object? is self.params.get("organization_id") just the id?

Reply:
:+1: Good catch, this needs to be the org instance